frontend/packages/kubevirt-plugin/integration-tests/tests/vmtemplate.wizard.scenario.ts

```diff
@@ -55,7 +55,6 @@ describe('Create VM from Template using wizard', () => {
             .setNamespace(testName)
             .setDescription(`VM from template ${vmt.name}`)
             .setFlavor(flavorConfigs.Tiny)
```

Comment:
we will still need to know the name of the template to create the VM from

Reply:
Will check this late.

Reply:
No need to use template here based on the actual test results.